go/vt/vttablet/tabletserver/vstreamer/vstreamer.go

```diff
@@ -643,7 +644,8 @@ func (vs *vstreamer) buildJournalPlan(id uint64, tm *mysql.TableMap) error {
 		return err
 	}
 	defer conn.Close()
-	qr, err := conn.ExecuteFetch("select * from _vt.resharding_journal where 1 != 1", 1, true)
+	qr, err := conn.ExecuteFetch(fmt.Sprintf("select * from %s.resharding_journal where 1 != 1",
+		sidecardb.GetIdentifier()), 1, true)
```

Comment:
should this use `sqlparser.BuildParsedQuery`

Reply:
Good catch! I somehow missed this one.